csharp/private/frameworks/netstandard21.BUILD

```diff
@@ -0,0 +1,971 @@
+# This file was generated by build\src\D2L.Build.BazelGenerator\FrameworkBuildGenerator\GenerateFrameworkBuildFile.cs
+# Don't edit it by hand! You should regenerate this if the .NET package gets updated.
+
+load("@d2l_rules_csharp//csharp:defs.bzl", "import_library")
+
+import_library(
+    name = "Microsoft.Win32.Primitives",
+    target_framework = "netstandard2.1",
+    refdll = "//:ref/netstandard2.1/Microsoft.Win32.Primitives.dll",
+    deps = ["//:netstandard"],
+    visibility = ["@net//:__pkg__"],
+)
+
+import_library(
+    name = "mscorlib",
+    target_framework = "netstandard2.1",
+    refdll = "//:ref/netstandard2.1/mscorlib.dll",
+    deps = ["//:netstandard"],

```

Comment:
Cool, I'll open an issue to make our `netstandard20` `BUILD` file look more like this one instead of the hacks I did.